src/scenes/AcceleratorRouter/Deliverables/DeliverableView.tsx

```diff
@@ -163,15 +179,25 @@ const DeliverableView = () => {
     [callToAction, optionsMenu]
   );
 
-  const crumbs: Crumb[] = useMemo(
-    () => [
-      {
-        name: activeLocale ? strings.DELIVERABLES : '',
-        to: APP_PATHS.ACCELERATOR_DELIVERABLES,
-      },
-    ],
-    [activeLocale]
-  );
+  const crumbs: Crumb[] = useMemo(() => {
+    if (!activeLocale) {
+      return [];
+    }
+
+    return source?.startsWith(APP_PATHS.ACCELERATOR_PROJECT_VIEW.replace(':projectId', ''))
+      ? [
```

Comment:
Approving, though will mention you could also add a new constant for this, like `ACCELERATOR_PROJECT_BASE` so that you don't have to do a replace here

Reply:
Good suggestion, will update.

Reply:
I opened a [PR in web-components](https://github.com/terraware/web-components/pull/661) that modifies the `Tabs` component to allow customizing tab styles. I'll remove this negative margin in a follow-up PR.